python/ray/_raylet.pyx

```diff
@@ -634,6 +634,13 @@ cdef execute_task(
             function = execution_info.function
 
             if core_worker.current_actor_is_asyncio():
+                if len(inspect.getmembers(
+                        actor.__class__,
+                        predicate=inspect.iscoroutinefunction)) == 0:
+                    raise RayActorError(
```

Comment:
Do we need to raise RayValueError instead? 
@kfstorm CC

Reply:
There is no RayValueError in our codebase, so I believe RayActorError does make sense.